socotratools/VERSION

```diff
@@ -1 +1 @@
-1.3.3
+1.3.4
```

Comment:
Should this change be here?

Reply:
Yes - updating public version as I iterate